features/node/kata/install_uninstall.feature

```diff
@@ -0,0 +1,41 @@
+Feature: kata related features
+  # @author pruan@redhat.com
+  # @case_id OCP-36508
+  @admin
+  @destructive
+  Scenario: kata container operator installation
+    Given the master version >= "4.6"
+    Given kata container has been installed successfully in the "kata-operator" project
+    And I verify kata container runtime is installed into the a worker node
+
+  # @author pruan@redhat.com
+  # @case_id OCP-36509
+  @admin
+  @destructive
+  Scenario: test delete kata installation
+    Given I remove kata operator from the namespace
+
+
+  Scenario: test install kata-webhook
```

Comment:
Is it a valid scenario ?

Reply:
@liangxia it will be, I just need to add the testcase into Polarion to get the proper Polarion testcase ID